src/random.cr

```diff
@@ -56,6 +56,14 @@ module Random
   # The integers must be uniformly distributed between 0 and the maximal value for the chosen type.
   abstract def next_u : UInt
 
+  # Generates a slice filled with *n* random bytes.
+  def random_bytes(n : Int) : Bytes
```

Comment:
@RX14 oh, I've considered doing this for performance, but in the end I think simplicity is more important. The consistency is with `SecureRandom.random_bytes`.

Anyway, I don't mind changing this, but I'm not sure if it's best. Leaving as is for now.